wordpress-mobile/AztecEditor-Android
# Issue/681 citation italic

### Fix
Fixes #681 by making a distinction between `<i>` and `<cite>`, to cope with how Gutenberg expects its `` block to be built.

### Test
1. Feed Aztec with the following code as described in the issue:
```

<blockquote class="wp-block-quote">
	<p>this </p>
	<p>is a</p>
	<p>quote</p><cite>this is a citation</cite></blockquote>

```

2. check that the visual representation respects the format as in:

<img width="435" alt="screen shot 2018-05-22 at 15 19 13" src="https://user-images.githubusercontent.com/6597771/40382005-a11a05f6-5dd3-11e8-9f4e-6268c2db632d.png">


3. toggle HTML mode again and verify the code `<cite>this is a citation</cite>` remained the same (i..e has not been converted to `<i>this is a citation</>`).

### Review
@daniloercoli @hypest
